Style, writing and formatting guide for standards

irisfaraway opened the pull request:
https://github.com/DEFRA/software-development-standards/issues/55

As discussed in the meeting today. This should cover the points we talked about, including Markdown and the concept of a standard being "easily testable".

This covers some basic formatting guidance, and some general writing advice for the tone and style of the guides. The goal is to make our standards easy to understand, and therefore easy to adopt.

I've also linked to some more extensive guidelines on writing for the web. These are written for a slightly different audience but I think the principles still apply. (Also, I wrote some of them back when I used to be a content person.)

irisfaraway commented:
I'm also not sure if this is the right folder for this – is a guide about writing guides too meta for /guides?

bensagar-ea commented:
> I'm also not sure if this is the right directory for this – is a guide about writing guides too meta for /guides?

Absolutely not - I whole-heartedly approve! 😃

bensagar-ea commented:
Overall very much a 👍 from me!

irisfaraway commented:
@bensagar-ea Cool, if you're happy with that location then I've updated the various tables of contents, and added it to CONTRIBUTING.md for good measure!
